openwisp_controller/connection/connectors/ssh.py

```diff
@@ -1,6 +1,6 @@
 import logging
 import socket
-import sys
+from StringIO import StringIO
```

Comment:
the python 3 version of that code is 'from io import StringIO`

Reply:
I messed up there sorry about that